spec/pronounce/data_reader_spec.rb

```diff
@@ -3,19 +3,18 @@
 
 module Pronounce
   describe DataReader do
-    let(:its) { DataReader }
+    subject(:reader) { DataReader }
 
     it 'loads the articulations file' do
-      expect(its.articulations).to include "AA\tvowel\n"
+      expect(reader.articulations['AA']).to eq 'vowel'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.